app/views/ops/_ldap_forest_entries.html.haml

```diff
@@ -19,7 +19,7 @@
         - if entry != "new"
           %tr
             %td
-              %button.btn.btn-default{:onclick => remote_function(:url => {:action => 'forest_select', :ldaphost_id => "new"}), :title => _("Click to add a new forest")}
+              %button.btn.btn-default{:type => 'button', :onclick => remote_function(:url => {:action => 'forest_select', :ldaphost_id => "new"}), :title => _("Click to add a new forest")}
```

Comment:
@skateman do we also need to add `:type` here https://github.com/ManageIQ/manageiq-ui-classic/blob/f84adbb3db34278a455a7f1fea43235eded1257e/app/views/ops/_ldap_forest_entries.html.haml#L74